src/main/java/duke/command/AddTaskCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.Ui;
```

Comment:
Consider sorting imports by lexicographical order. Noticed this in a couple of areas as well.